integration/fixtures/test-images.json

```diff
@@ -1,188 +1,10 @@
 [
     {
-        "image": "docker.io/grafana/grafana",
```

Comment:
i am guessing this was for testing. if this is ready, can you add this back please?